stingray/tests/test_powerspectrum.py

```diff
@@ -184,7 +181,7 @@ def rebin_several(self, df):
         """
         ps = Powerspectrum(lc=self.lc, norm="Leahy")
         bin_ps = ps.rebin(df)
-        assert np.isclose(bin_ps.freq[0], bin_ps.df, atol=1e-4, rtol=1e-4)
+        assert np.isclose(bin_ps.freq[1], bin_ps.df, atol=1e-4, rtol=1e-4)
```

Comment:
I changed the rebin method on powerspectrum as @dhuppenkothen suggested but the tests failed then. I changed the assertion test in the function `tests.test_powerspectrum.rebin_several()'

Can someone else, please, check what is the expected behavior with the new frequency array?